src/interface/src/app/treatments/direct-impacts/direct-impacts.component.ts

```diff
@@ -116,6 +125,25 @@ export class DirectImpactsComponent implements OnInit, OnDestroy {
 
   breadcrumbs$ = this.treatmentsState.breadcrumbs$;
   treatmentPlan$ = this.treatmentsState.treatmentPlan$;
+  activeStand$ = this.directImpactsStateService.activeStand$;
+
+  selectedChartProjectArea$ =
+    this.directImpactsStateService.selectedProjectArea$;
+  showTreatmentPrescription = false;
+  changeChartButtons: PanelIconButton[] = [
+    { icon: 'open_in_full', actionName: 'expand' },
+  ];
+
+  availableProjectAreas$ = this.treatmentsState.summary$.pipe(
+    map((summary) => {
+      // TODO: can remove this, in favor of using natsort on backend,
```

Comment:
Yeah , I think @george-silva mentioned this at some point but not sure if we created a story for it. Can you create one so we can track?

Reply:
added https://sig-gis.atlassian.net/browse/PLAN-1885